docs/en/api/route-object.md

```diff
@@ -10,15 +10,15 @@ The route object can be found in multiple places:
 
 - As the return value of calling `router.match(location)`;
 
-- Inside navigation guards as the first argument:
+- Inside navigation guards as the first and second argument:
```

Comment:
Well I guess it should be `as the first two arguments`.
https://www.quora.com/Which-one-is-grammatically-correct-The-first-and-the-second-chapter-is-very-difficult-or-The-first-and-second-chapters-are-very-difficult